### DISCUSSION TITLE
Plot file needed for anything?

### DISCUSSION THREAD

**Discussion by cannikin** [nekno/DV7toDV8#12] [Q&A] [answered]
I noticed that even without `--keep-files` the .png plot remains after the script is done. Is that needed for anything after the fact (like to mux the DV7 EL back into the original file) or is that just for info purposes?

**Comment by nekno** [comment, marked as answer]
That's just for info purposes so you can see if the mastering info looks off, or if the graph shows a straight line, in which case the metadata is basically useless. The file will trigger DV mode on your display, but only be rendering static HDR10 metadata, so you might as well discard the RPU altogether and watch in HDR10 to ensure an accurate picture. Unfortunately, some major releases have been produced that way.
